### PR TITLE
Accept more than one possible current channel for auxiliary charms

### LINKED ISSUE #92: Accept more than one possible channel for RabbitMQ and hacluster
There is a discrepancy between the [charm docs](https://docs.openstack.org/charm-guide/latest/project/charm-delivery.html) and deployed clouds. It's possible to have rabbitmq and hacluster using channels for newer OpenStack releases. E.g:
- documentation says that  rabbitmq-server on focal should track channel 3.8 for ussuri until yoga. However it's relatively common to find  clouds tracking channel 3.9
- documentation says that  haclusterr on focal should track channel 2.0.3 for ussuri until yoga. However it's relatively common to find  clouds tracking channel 2.4

### PR DESCRIPTION
**Pull request by gabrielcocenza**
- rabbitmq and hacluster can have more than one compatible channel
  for a range of OpenStack releases.
  
  Closes #92

### CODE REVIEW

File: cou/apps/app.py [hunk 1]
```diff
@@ -487,19 +485,23 @@ def _get_refresh_charm_plan(
         :rtype: Optional[UpgradeStep]
         """
         switch = None
-        description = (
-            f"Changing '{self.name}' channel from: '{self.channel}' "
-            f"to: '{self.expected_current_channel}'"
-        )
```
Comment (agileshaw) [inline cou/apps/app.py:493]: A default description is still needed at L#514 for charmhub aoplication with channel outside of the possible_current_channels list.

Comment (agileshaw) [reply cou/apps/app.py:493]: Also, could you add a unit test for this case?

**Review by agileshaw — CHANGES_REQUESTED** [review]

**Review by agileshaw — APPROVED** [review]
LGTM

**Review by rgildein — APPROVED** [review]
LGTM

**Review by agileshaw — APPROVED** [review]

**Pull request merged**